### DISCUSSION TITLE
Support for Lottie Files

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [axmolengine/axmol#1892] [Q&A] [answered]
Hi,

There is any support for Lottie Files?

Thanks.

**Comment by halx99** [comment]
What do you mean?

**Reply by paulocoutinhox** [reply]
Lottie Files is an animation in json high used in mobile apps:
https://lottiefiles.com/

<img width="793" alt="image" src="https://github.com/axmolengine/axmol/assets/395096/f89f5230-562e-466a-8d99-4c8cf0e4ce81">

I don't find any cocos2d-x compatible Lottie Files player :(

You can see it working on my site:
https://devxz.com/lottie-player/

**Reply by paulocoutinhox** [reply]
There is something for Axmol?

**Comment by Arvant** [comment]
in unity use https://github.com/Samsung/rlottie for do it.

**Reply by aismann** [reply]
Axmol (and cocos2dx) supports no native animation file formats like gif, lottie, etc.
Everybody can write a PR to change this.
[Edit]: It seems not realy an important feature?

**Comment by DelinWorks** [comment, marked as answer]
I think it's out of scope for axmol, if you need this feature then I think that simple vertex animation shaders and skinning (theres also spine) will do the job perfectly, Lottie is mainly used for OS UI systems and web like you did, not native low level game engines

**Reply by halx99** [reply]
> I think it's out of scope for axmol, if you need this feature then I think that simple vertex animation shaders and skinning will do the job perfectly, Lottie is mainly used for OS UI systems and web like you did, not native low level game engines

Agree, closed

**Comment by iAndyHD3** [comment]
Alternatively: separate repo extension with support is always welcome
